code/game/machinery/doors/mineral.dm

```diff
@@ -224,6 +224,11 @@
 		add_fingerprint(user)
 		SwitchState()
 
+/obj/machinery/door/mineral/resin/bullet_act(var/obj/item/projectile/Proj)
+	if(Proj.damage_type == BRUTE || Proj.damage_type == BURN)
```

Comment:
does the damage type really matters

Reply:
I opted to be conservative since destroying it with a decloner is arguably harmful to emulsiuons.